tools/admin-console/templates/healthauthority.html

```diff
@@ -55,6 +55,17 @@
         </small>
       </div>
 
+      <div class="form-group">
+        <label for="enable-stats-api">Enable Stats API Access</label>
+        <select name="enable-stats-api" id="enable-stats-api" class="form-control custom-select">
```

Comment:
This is probably better served as a checkbox, but meh.

Reply:
we use select a lot for this...